capstone/src/main/resources/static/script.js

```diff
@@ -158,7 +157,6 @@ function getUserType() {
 function createProfile(data, isOrganization) {
```

Comment:
Might be more readable if you pass the user, not the data so you don't have to write data[0] each time

Reply:
Good point probably would make the code clearer too.